### ISSUE TITLE
Is there any way to explicitly set the initial ordering priority?

### ISSUE THREAD

**Issue by jariberi** [vaadin/vaadin-grid#1183]
Is there any way to explicitly set the initial ordering priority in vaadin grid columns?

**Comment by web-padawan** [comment]
You can set the `direction` attribute on the `<vaadin-grid-sorter>` element. But actually the sorters array is populating in reverse direction because of [this line](https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid-sort-mixin.html#L70). See this [jsfiddle](https://jsfiddle.net/webpadawan/99wzv9vm/) for demo and note how the "First name" order changes once you click on the correspondent header cell.

Do I understand correctly that you expect the API to be able to configure how these values of the sorters are getting applied by default? Would you expect it to be a numeric index or something different?

**Comment by jariberi** [comment]
Yes, a numeric index would be great. Maybe in each vaadin-grid-sorter tag and the order like a normal JS array...

**Comment by karsten-eger** [comment]
I stumbled across this issue as well last week. Would be nice to have an option for this. The priority could even be the index in the sortOrder array.

**Comment by HJK181** [comment]
It also would be nice to have the possibility to control which sorting direction will be applied on the first click of a column. So the column is not sorted initially but on first click I want it to be sorted in desc order not in asc order.

I don't see any possibility to achieve this yet.

**Comment by jariberi** [comment]
Any update?

**Comment by sgoodrow** [comment]
I would also like this, but I think the sorting algorithm would need to be changed for both `multi sort` and `single sort` because of the way initialization occurs on each column. That, or an added hook after connection is done which allows specification of default sort orders.

The problem is that the sort algorithm re-assigns the order based on the array index, and adding elements to the array takes place in a static order (column order), so the default sort order would always be the column order.

**Comment by fireplace009** [comment]
### Any update about this ?
I have the same request, after a page refresh, the ordering is no longer displayed correctly.
If we could specify the order just like we can with the direction. Please add asap :/

For example:
<vaadin-grid-sorter on-tap="_sort" direction="[[_direction(header.name)]]" **order="[[_order(header)]]"** path="[[header.name]]" >


### Bug screenshot:

My grid is showing like this
![image](https://user-images.githubusercontent.com/19831551/70719599-97a41e00-1cf2-11ea-99ff-9ad696ab7fc4.png)

Then the user does a page Refresh (F5)
![image](https://user-images.githubusercontent.com/19831551/70719649-aab6ee00-1cf2-11ea-8b6c-cc0218256528.png)

(the direction is correct, the sort order is not)
--------
